autotx/utils/constants.py

```diff
@@ -0,0 +1,5 @@
+import os
+
+OPENAI_MODEL_NAME = os.environ.get("OPENAI_MODEL_NAME", "gpt-4-turbo-preview")
+OPENAI_API_KEY = os.environ.get("OPENAI_API_KEY", None)
```

Comment:
I think we've regressed this feature: https://github.com/polywrap/AutoTx/pull/123

Reply:
i tested with llama2 (using ollama) - just by setting the needed variables in the `.env` it works with local llms